src/drivers/imu/adis16497/ADIS16497.cpp

```diff
@@ -53,11 +53,15 @@ static constexpr uint8_t CONFIG  	= 0x0A; // Control, clock and miscellaneous co
 static constexpr uint8_t DEC_RATE  	= 0x0C; // Control, output sample rate decimation
 static constexpr uint8_t NULL_CNFG  	= 0x0E; // Control, automatic bias correction configuration
 static constexpr uint8_t SYNC_SCALE  	= 0x10; // Control, automatic bias correction configuration
-static constexpr uint8_t RANG_MDL   	= 0x12; // Measurement range (model-specific) Identifier TODO use this
+static constexpr uint8_t RANG_MDL   	= 0x12; // Measurement range (model-specific) identifier
 static constexpr uint8_t FILTR_BNK_0   	= 0x16; // Filter selection
 static constexpr uint8_t FILTR_BNK_1   	= 0x18; // Filter selection
 
-static constexpr uint16_t PROD_ID_ADIS16497 = 0x4071;	// ADIS16497 Identification, device number
+static constexpr uint16_t PROD_ID_ADIS16497 = 0x4071; // ADIS16497 device number
+
+static constexpr uint16_t RANG_MDL_1BMLZ = 0x03; // ADIS16497 model number
```

Comment:
The comments are generic and repeated. You could make them match the datasheet.
```suggestion
static constexpr uint16_t RANG_MDL_1BMLZ = 0b0011; // ADIS16497-1 (±125°/sec)
```